docs/planning/bug_triage.md

```diff
@@ -0,0 +1,44 @@
+# Knative Eventing Bug Triage and Management
+
+This document summarizes how bugs will be triaged and managed in Knative Eventing WG.
+This is a supplement to [Sprint planning](sprintplanning.md) process.
+
+## Goal
+
+- Keep a low bug count. Bug count should not be consistently increasing release over release esp [priority/important-longterm](#Bug-priorities).
+- Process shouldn't be taxing for a single person or the team.
+- Maintain high product quality.
+
+## Bug priorities
+
+- priority/critical-urgent: Bugs that need to be fixed now, and possibly cherrypicked into the most recent release. May need patch-fix.
+- priority/important-soon: Bugs that need to be fixed in the next release cycle, and but do not need a patch-fix.
+- priority/important-longterm: Bugs that need to be fixed, and will be fixed in next 2-3 releases.
+- priority/awaiting-more-evidence: Bugs which might need to be fixed, but need more details first.
+
+## How to manage bugs?
+
+- Submit new bugs anytime by creating a new issue of type “Bug”.
+- Please fill the bug template and optionally propose a priority.
+- All new bugs will be labelled “kind/bug” as part of the template
+ Bugs will be triaged weekly (this could be modified based on incoming bugs
```

Comment:
Team should not wait for the meeting if it's issues that are easy to handle.
Maybe we could add a label `triage` for issues that we need to quickly discuss in the next triage meeting, in the meeting this are the first to address and the label `triage` will remove during the meeting as they are discuss.

Reply:
The process doesn't stop someone from fixing a non-triaged bug. However if there are triaged and untriaged bugs then team should prioritize the bugs that are triage and marked for current release. Hence not creating any special label. If there is a new bug that is not assigned to anyone then it should be triaged.